src/main/java/uk/gov/hmcts/ccd/domain/service/common/AccessControlService.java

```diff
@@ -221,7 +222,15 @@ private JsonNode filterChildrenUsingJsonNode(final CaseFieldDefinition caseField
                                                  final Set<String> userRoles,
                                                  final Predicate<AccessControlList> access,
                                                  boolean isClassification) {
-        if (caseField.isCompoundFieldType()) {
+        // We are getting class com.fasterxml.jackson.databind.node.MissingNode cannot be cast to class
+        // com.fasterxml.jackson.databind.node.ObjectNode exception when childField.getId() is null, so this
+        // is added to check if the value exists.
+        if (jsonNode instanceof MissingNode) {
+            LOG.info("Can not found value for  jsonNode={}, jsonNodeParent={}, user roles={}, caseType={}, "
+                    + "accessControlList={}",
+                 jsonNode, jsonNode.findParent("id"), userRoles, caseField.getCaseTypeId(),
```

Comment:
jsonNode.findParent("id") always return a null for MissingNode. Is there any value to log this?

Reply:
We're thinking to replace it with the full path to the missing node. That part will follow later subject to other priorities and in the worst case, we can remove parent log as we already have enough on this log line combined with other telemetries.